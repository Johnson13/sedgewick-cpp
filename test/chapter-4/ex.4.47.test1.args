EAs+Y+QUE**+st+*+IO*n++*
