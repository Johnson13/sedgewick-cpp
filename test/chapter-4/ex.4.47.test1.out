E     E
A     A E
s     A E s
+  A  E s
Y     Y E s
+  Y  E s
Q     Q E s
U     U Q E s
E     E U Q E s
*  s  E U Q E
*  E  E U Q
+  E  U Q
s     U Q s
t     U Q s t
+  U  Q s t
*  t  Q s
+  Q  s
I     I s
O     O I s
*  s  O I
n     O I n
+  O  I n
+  I  n
*  n  
